modules/planning/tasks/optimizers/piecewise_jerk_speed/piecewise_jerk_speed_optimizer.cc

```diff
@@ -0,0 +1,152 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file piecewise_jerk_fallback_speed.cc
+ **/
+
+#include "modules/planning/tasks/optimizers/piecewise_jerk_speed/piecewise_jerk_speed_optimizer.h"
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "modules/common/proto/pnc_point.pb.h"
+
+#include "modules/common/vehicle_state/vehicle_state_provider.h"
+#include "modules/planning/common/planning_gflags.h"
+#include "modules/planning/common/st_graph_data.h"
+#include "modules/planning/math/piecewise_jerk/path_time_qp_problem.h"
+
+namespace apollo {
+namespace planning {
+
+using apollo::common::ErrorCode;
+using apollo::common::Status;
+using apollo::common::TrajectoryPoint;
+
+PiecewiseJerkSpeedOptimizer::PiecewiseJerkSpeedOptimizer(
+    const TaskConfig& config)
+    : SpeedOptimizer(config) {
+  SetName("PiecewiseJerkSpeedOptimizer");
+  // CHECK(config_.has_piecewise_jerk_speed_config());
```

Comment:
Remove.

Reply:
This will be enable when the config is ready